articles/server-platforms/rails/05-linking-accounts.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Linking Accounts
+description: Link and Unlink accounts from other identity providers using the Rails SDK.
+---
+
+## Ruby On Rails - Linking Accounts
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+* Ruby 2.3.1
+* Rails 5.0.0
+:::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'omniauth-auth0',
+  pkgBranch: 'master',
+  pkgPath: 'examples/ruby-on-rails-webapp',
+  pkgFilePath: null,
+  pkgType: 'server'
+}) %>
+
+/_ TODO: REMOVE THIS line
+
+In this tutorial you'll learn how to use the Auth0 Rails SDK to explicitly link user accounts from different identity providers, allowing a user to authenticate from any of their accounts and still be recognized by your app and associated with the same user profile.
+
+The sample application allows the users to log in with an initial provider, and provides a form that enables them to link another account to the first one using Auth0 Lock. Click [here](/link-accounts) to learn more about account linking.
+
+### 1. Add dependencies
+
+Add the following dependencies to your `Gemfile` and run `bundle install`
+
+${snippet(meta.snippets.dependencies)}
+
+### 2. Handling the second authentication with Lock
+In [Login](/quickstart/webapp/rails/01-login), you created a js function to handle the authentication with Lock. You'll use the same pattern for your application to trigger the second authentication.
+
+Add the following method to `/app/assets/javascripts/home.js.rb`:
+
+```js
+function linkPasswordAccount(){
+	connection = $("#link_provider").val();
+  var linkOptions = {
+		auth: {
+			redirectUrl: '${' <%= Rails.application.secrets.auth0_callback_url %> '}',
```

Comment:
I'm not familiar with this `'${'` syntax - is this part of our docs preprocessor?